src/main/java/io/codelex/flightplanner/config/DatasourceConfiguration.java

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Tev vajag šito aizkomentēto klasi? Ja ne, tad dzēs ārā.

Reply:
Tam tur vispār būt nevajadzēja. Laikam biju aizmirsis nopushot versiju bez.